README.md

```diff
@@ -128,37 +128,36 @@ This year's baseline is a ensembled neural transition system based on the
 imitation learning paradigm introduced by Makarov & Clematide (2018). A variant
 of this system (Makarov & Clematide 2020) was the second-best system overall in
 the 2020 shared task (Gorman et al. 2020). Code for the baseline library can be
-found in the [`baseline`](baseline) directory. Baseline development set results
-are given below.
-
-|                        |  WER (dev) |
```

Comment:
What's with number changes here? (the Bulgarian tab fix is good of course.)

Reply:
That's weird. I didn't touch the README at all. Possibly old edit that slipped through earlier in the pipeline?

Reply:
Yeah, please revert if so.